rpcs3/Emu/RSX/GL/rsx_gl_texture.cpp

```diff
@@ -469,11 +498,6 @@ namespace rsx
 			glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MIN_FILTER, gl_tex_min_filter[tex.min_filter()]);
 			glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MAG_FILTER, gl_tex_mag_filter[tex.mag_filter()]);
 			glTexParameterf(GL_TEXTURE_2D, GL_TEXTURE_MAX_ANISOTROPY_EXT, max_aniso(tex.max_aniso()));
-
-			if (is_swizzled && format == CELL_GCM_TEXTURE_A8R8G8B8)
-			{
-				free(unswizzledPixels);
```

Comment:
Originally, only A8R8G8B8 had swizzling enabled, and moving the free back up to the switch block caused crashing for some reason. I did not implement that myself. The original implementer might give better insight here.